src/XMakeTasks/Resources/Strings.resx

```diff
@@ -2085,6 +2085,10 @@
     <value>MSB3491: Could not write lines to file "{0}". {1}</value>
     <comment>{StrBegin="MSB3491: "}</comment>
   </data>
+  <data name="WriteLinesToFile.ErrorReadingFile">
+    <value>MSB3492: Could not read existing file "{0}" to compare contents against. Will attempt to overwrite only.</value>
```

Comment:
`Could not read existing file "{0}" to determine whether its contents are up to date. Overwriting it.` ?